src/demo/DlToastMessageDemo.vue

```diff
@@ -0,0 +1,164 @@
+                        v-model="type"
+                        value="info"
+                        label="Info"
+                    />
+                </div>
+                <div>
+                    Position
+                    <dl-radio
+                        v-model="position"
+                        value="bottom"
+                        label="Bottom"
+                    />
+                    <dl-radio
+                        v-model="position"
+                        value="bottom-left"
+                        label="Bottom left"
+                    />
+                    <dl-radio
+                        v-model="position"
+                        value="bottom-right"
+                        label="Bottom right"
+                    />
+                    <dl-radio
+                        v-model="position"
+                        value="top"
+                        label="Top"
+                    />
+                    <dl-radio
+                        v-model="position"
+                        value="top-left"
+                        label="Top left"
+                    />
+                    <dl-radio
+                        v-model="position"
+                        value="top-right"
+                        label="Top right"
+                    />
+                </div>
+                <div>
+                    <dl-switch
+                        v-model="activeDuration"
+                        left-label="Active duration"
+                    />
+                    <hr>
+                    <dl-switch
+                        v-model="closable"
+                        left-label="Closable"
+                    />
+                </div>
+            </div>
+        </div>
+        <dl-button @click="showToastMessage">
+            Show Toast Message
+        </dl-button>
+    </div>
+</template>
+
+<script lang="ts">
+import { DlButton } from '../components'
+import { useDlToast } from '../components/DlToastMessage'
+import { defineComponent, ref } from 'vue-demi'
+import DlTextInput from '../components/DlTextInput.vue'
+import DlRadio from '../components/DlRadio.vue'
+import DlTextArea from '../components/DlTextArea.vue'
+import DlSwitch from '../components/DlSwitch.vue'
+export default defineComponent({
+    name: 'DlToast',
+    components: {
+        DlSwitch,
+        DlTextArea,
+        DlRadio,
+        DlTextInput,
+        DlButton
+    },
+    setup() {
+        const message = ref(
+            'This is a sample message with a <a href="https://google.com/">link</a>'
+        )
+        const duration = ref(10)
+        const type = ref('success')
+        const position = ref('bottom')
+        const spaceBetween = ref(10)
+        const indentFromScreenBorder = ref(10)
+        const activeDuration = ref(true)
+        const closable = ref(true)
+        const lineHeight = ref(18)
+        function showToastMessage() {
+            useDlToast.open({
```

Comment:
please change it to DlToast instead of useDlToast 
it will be exported to our users so we need it to be intuitive like components